core/src/main/java/io/questdb/std/Files.java

```diff
@@ -71,9 +71,9 @@ public static synchronized boolean auditClose(long fd) {
         return true;
     }
 
-    public static synchronized boolean auditOpen(long fd) {
-        if (null == openFds) {
-            openFds = new LongHashSet();
+    public static synchronized boolean auditOpen(int fd) {
```

Comment:
on windows, `fd` defined as `HANDLE`, which is a `void *`. This cannot be `int` unless i'm missing something ?